__tests__/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity.test.ts

```diff
@@ -0,0 +1,115 @@
+import { awardConnectExternalAccountActivityPoints, ConnectExternalAccountActivity } from "../../../src/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity";
+
+jest.mock('@careevolution/mydatahelps-js', () => {
+
+    const externalAccounts = [{
+        "id" : 1,
+        "provider": { "id": 11, "name": "Garmin", "category": "Device Manufacturer"},
+        status: "fetchComplete"
+    },
+    {
+        "id" : 2,
+        "provider": { "id": 22, "name": "Fitbit", "category": "Device Manufacturer"},
+        status: "fetchComplete"
+    },
+    {
+        "id" : 3,
+        "provider": { "id": 11, "name": "FitBit", "category": "Device Manufacturer"} ,
```

Comment:
Maybe use an `id` of 33 here to avoid any potential collisions with the Garmin provider.

Reply:
Actually, I just realized this is a second Fitbit provider.  Was that intentional?  A don't think a user should be able to have two Fitbit connections, but if they did, the provider would have the same id.

Reply:
I updated the 3rd external accounts provider id to reflect the desired mock data - multiple external accounts connected to the same providers (Fitbit/Garmin/Etc). 
I have updated the provider id to match the fitbit provider